dask_sql/context.py

```diff
@@ -805,7 +807,11 @@ def _get_ral(self, sql):
                 f"Multiple 'Statements' encountered for SQL {sql}. Please share this with the dev team!"
             )
 
-        nonOptimizedRel = self.context.logical_relational_algebra(sqlTree[0])
+        try:
+            nonOptimizedRel = self.context.logical_relational_algebra(sqlTree[0])
+        except DFParsingException as pe:
+            raise ParsingException(sql, str(pe)) from None
```

Comment:
If one doesn't exist already we should add a test case checking for this.

Reply:
Several already exist one coming top of mind is the `test_wrong_input` pytest in `test_select.py`